docs/API.md

```diff
@@ -382,6 +382,17 @@ An array of *Profile names* in lower case. `"default"` is always included.
 Returns all valid [language tags][language tag] generated from the setting in
 the configuration file.
 
+The `language tags` are generated from the configured [locale tags].
+Each `locale tag` will generate two `language tags`, a short tag
+equal to the first two letters (usually the same as a language
+code) and a long tag which is equal to the full `locale tag`.
+If "en_US" is the `locale tag` then "en" and "en_US" are the
+`language tags`.
+
+If there are two `locale tags` that would give the same short
+`language tag` then that is excluded. E.g. "en_US en_UK" will
+only give "en_US" and "en_UK" as `language tags`.
```

Comment:
Lines 385 to 394, in your proposal, should rather be integrated into the language tag section that this section already refers to. You have a reference to "locale tag" but no link.

Reply:
I fixed the link and tweaked the language a little bit.

I'm not sure I understand what your mean about integrating. But let me take this moment to explain why I moved these paragraphs here.

One way to to draw the line between the RPCAPI and the configuration is so that the RPCAPI is about "doing" and the configuration is about "being". From this perspective "generating" is a form of doing, and so it belongs in the RPCAPI. The configuration on the other hand just sits there quite inert concerning itself only with being valid and having a meaning. I believe this perspective is a good guiding principle that is fairly easy to understand and apply.

Reply:
There are two sections in the API document on language tags. You have put the text under the `get_language_tags`method section. I think it is better to have all details under the "language tag" section and then referring to that.

Reply:
Oh, I see. Yeah. I'll make an update.